_config.yml

```diff
@@ -1,21 +1,121 @@
+search_enabled: true
+search:
+  # Split pages into sections that can be searched individually
+  # Supports 1 - 6, default: 2
+  heading_level: 2
+  # Maximum amount of previews per search result
+  # Default: 3
+  previews: 3
+  # Maximum amount of words to display before a matched word in the preview
+  # Default: 5
+  preview_words_before: 5
+  # Maximum amount of words to display after a matched word in the preview
+  # Default: 10
+  preview_words_after: 10
+  # Set the search token separator
+  # Default: /[\s\-/]+/
+  # Example: enable support for hyphenated search words
+  tokenizer_separator: /[\s/]+/
+  # Display the relative url in search results
+  # Supports true (default) or false
+  rel_url: true
+  # Enable or disable the search button that appears in the bottom right corner of every page
+  # Supports true or false (default)
+  button: false
+
+# Enable or disable heading anchors
+heading_anchors: true
+
+# Aux links for the upper right navigation
+aux_links:
+  "Konveyor Move2Kube":
+    - "//github.com/konveyor/move2kube"
+
+# Makes Aux links open in a new tab. Default is false
+aux_links_new_tab: false
+
+# Sort order for navigation links
+nav_sort: case_insensitive # default, equivalent to nil
+# nav_sort: case_sensitive # Capital letters sorted before lowercase
+
+# Footer content
+# appears at the bottom of every page's main content
+
+# Back to top link
+back_to_top: true
+back_to_top_text: "Back to top"
+
+#footer_content: "<a href=\"https:konveyor.github.io/move2kube\">Move2Kube</a> is maintained by <a href=\"https://konveyor.github.io\">Konveyor.</a><br>Copyright &copy; 2020 IBM Corporation. All right reserved."
+
+# Footer last edited timestamp
+last_edit_timestamp: true # show or hide edit time - page must have `last_modified_date` defined in the frontmatter
+last_edit_time_format: "%b %e %Y at %I:%M %p" # uses ruby's time format: https://ruby-doc.org/stdlib-2.7.0/libdoc/time/rdoc/Time.html
+
+
+
+# Footer "Edit this page on GitHub" link text
+gh_edit_link: false # show or hide edit this page link
+gh_edit_link_text: "Edit this page on GitHub"
+gh_edit_repository: "https://github.com/konveyor/move2kube" # the github URL for your repo
+gh_edit_branch: "gh-pages" # the branch that your docs is served from
+gh_edit_view_mode: "tree" # "tree" or "edit" if you want the user to jump into the editor immediately
+
+# Color scheme currently only supports "dark", "light"/nil (default), or a custom scheme that you define
+color_scheme: konveyor
+
+# External Links
+external_navigation:
+  - url: https://kubernetes.slack.com/archives/CR85S82A2
+    title: "&nbsp;&emsp;&emsp;Slack"
+  - url: https://konveyor.github.io/
+    title: "&nbsp;&emsp;&emsp;Konveyor"
+  - url: https://groups.google.com/g/konveyorio
+    title: "&nbsp;&emsp;&emsp;Forum"
+  - url: https://github.com/konveyor/move2kube
+    title: "&nbsp;&emsp;&emsp;View on GitHub"
+
+# Google Analytics Tracking (optional)
+# e.g, UA-1234567-89
+ga_tracking: UA-2709176-10
```

Comment:
Where did you get this id from? We should use the Konveyor repo google account

Reply:
Oh, it came by default from the `just-the-docs` theme. Please suggest if I should I change it to `G-J08VDXBTDB` which is mentioned in the _config.yml of konveyor.github.io or should I comment this part for now?
<img width="725" alt="Screenshot 2020-11-10 at 6 46 57 PM" src="https://user-images.githubusercontent.com/14867323/98678797-1f702d80-2385-11eb-97d5-845828f1f529.png">

Reply:
Yes, use "G-J08VDXBTDB"

Reply:
Yes sure, I have updated it.